lib/log/RolarProgressLog.ts

```diff
@@ -123,13 +122,23 @@ export class RolarProgressLog implements ProgressLog {
                     method: HttpMethod.Post,
                     body: {
                         host: os.hostname(),
-                        content: postingLogs,
+                        content: postingLogs || [],
                     },
                     headers: { "Content-Type": "application/json" },
+                    retry: {
+                        retries: 0,
```

Comment:
Why no retry?

Reply:
Because each error will add the entries back into the instance variable and it will be retried after the timeout anyway.